a11y-input/test/ally-input.test.js

```diff
@@ -72,6 +72,8 @@ describe('a11y input', () => {
       <a11y-input .value=${'foo'}></a11y-input>
     `));
 
+    // add debugger
+    debugger;
```

Comment:
* This did not catch the error, which means the failure **isn't** the result of this test.
* So, `debugger` needs to be added to `a11y-input/src/a11y-input.js`.